src/control/server/ctl_storage_rpc.go

```diff
@@ -50,6 +50,38 @@ func newState(log logging.Logger, status ctlpb.ResponseStatus, errMsg string, in
 	return state
 }
 
+func translateModules(mms []scm.Module) (pbMms types.ScmModules) {
```

Comment:
Minor naming nit - maybe something like translateModulesToPb would make clear in the name what we're translating from/to. It's obvious in the declaration but I'm thinking in usage.
Same thing on translateNamespaces below.

Reply:
yes that would be consistent with SystemMemberToPB